src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1124,10 +1126,10 @@ sub check_star_catalog {
 	# Faint and bright limits ~ACA-009 ACA-010
 	if ($type ne 'MON' and $mag ne '---') {
 
-	    if ($mag < $mag_bright or $mag > $mag_faint_red) {
+	    if ($mag < $mag_bright or $mag > $self->{mag_faint_red}) {
```

Comment:
I had some text in
https://github.com/sot/starcheck/commit/e463a46992a58e681175198a448b3e7e2a0c554c
that should work.